engines/dfc_provider/spec/controllers/dfc_provider/api/products_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe DfcProvider::Api::ProductsController, type: :controller do
+  render_views
+
+  let(:user) { create(:user) }
+  let(:enterprise) { create(:distributor_enterprise, owner: user) }
+  let(:product) do
+    create(:simple_product, supplier: enterprise )
+  end
+  let!(:visible_inventory_item) do
+    create(:inventory_item,
+           enterprise: enterprise,
+           variant: product.variants.first,
+           visible: true)
+  end
+
+  describe('.index') do
+    context 'with authorization token' do
+      before do
+        request.env['Authorization'] = 'Bearer 123456.abcdef.123456'
+      end
+
+      context 'with an authenticated user' do
+        before do
+          allow_any_instance_of(DfcProvider::AuthorizationControl)
+            .to receive(:process)
+            .and_return(user)
+        end
+
+        context 'with an enterprise' do
+          before { get :index }
+
+          it 'is successful' do
+            expect(response.status).to eq 200
+          end
+
+          it 'renders the related product' do
+            expect(response.body)
+              .to include("\"DFC:description\":\"#{product.variants.first.name}\"")
```

Comment:
If this test is just checking that the specific product has been included by making sure it's name is present in the json (and not testing the whole json output formatting generally), we could just do:
`.to include product.variants.first.name`
or 
`.to match product.variants.first.name`

Reply:
Ok, why not.